src/client/state/actions/user.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
+import {
+  ADD_USERS,
+  EMPTY_USERS,
+  START_LOADING_USERS,
+  STOP_LOADING_USERS,
+  SET_CURRENT_USER,
+  EMPTY_CURRENT_USER
+} from "state/actions/types";
+
+export const addUsers = users => {
+  return {
```

Comment:
To keep these more concise, you could set these up like this: 

```
export const addUsers = users => ({type: ADD_USERS, users});
```

Reply:
👍